profiles/common.nix

```
@@ -14,6 +14,10 @@
   # Allow PMTU / DHCP
   networking.firewall.allowPing = true;
 
+  # Use networkd instead of the pile of shell scripts
+  networking.useNetworkd = true;
+  networking.useDHCP = false;
```

Could be dangerous to overwrite defaults that people are already used to.

```suggestion
  networking.useNetworkd = lib.mkDefault true;
  networking.useDHCP = lib.mkDefault false;
```